MapboxNavigation/Resources/nl.lproj/Localizable.strings

```diff
@@ -44,7 +44,7 @@
 "RESUME" = "Doorgaan";
 
 /* The text of a banner that appears during turn-by-turn navigation when route simulation is enabled. */
-"USER_IN_SIMULATION_MODE" = "Navigatie Simuleren";
+"USER_IN_SIMULATION_MODE" = "Simulating Navigation at %d×";
```

Comment:
Shouldn't this be `"Navigatie Simuleren op %dx"`?

Reply:
It should but this string has not yet been translated in Transifex.